package.json

```diff
@@ -4,7 +4,7 @@
   "keywords": [
     "assert"
   ],
-  "version": "1.3.0",
+  "version": "1.4.0",
```

Comment:
Hey sorry that version bump is from the previous publish that I hadn't
pushed to master, as for the version of buffer, my bad will fix

On Fri, May 20, 2016, 6:44 PM Roman Shtylman <notifications@github.com>
wrote:

> In package.json
> <https://github.com/defunctzombie/commonjs-assert/pull/22#discussion_r64121510>
> :
>
> > @@ -4,7 +4,7 @@
> >    "keywords": [
> >      "assert"
> >    ],
> > -  "version": "1.3.0",
> > +  "version": "1.4.0",
>
> PRs shouldn't have version bumps, do those separately. This allows you to
> merge several PRs without deciding that the version change is yet.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/defunctzombie/commonjs-assert/pull/22/files/e502eca15424e3118e0e01ffad3e057b274cfd13#r64121510>
>